### PR TITLE
[FR] Add  `Client/Release stream/Lazer/Feature comparison`

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+Les beatmaps convertis du mode de jeu osu! en osu!mania avaient un ensemble de fenêtres de jugement qui n'étaient pas affectées par l'OD.
+
+|  |  |
+| :-- | :-: |
+| Rupture de la rétrocompatibilité | Oui |
+| Support du retour au mod classique | Non |
+| Modifié intentionnellement | Oui |
+| Nécessité d'une réflexion plus approfondie | No? |
+
+#### Les ajustements du débit audio affectent les fenêtres de jugement dans osu!mania
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:359]:
```suggestion
#### Les ajustements de la vitesse des beatmaps affectent les fenêtres de jugement dans osu!mania
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+
+Sur stable, osu!mania annule les modifications apportées aux fenêtres de jugement par l'utilisation du mod DT/HT.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:361]:
```suggestion
Sur stable, osu!mania annule les modifications apportées aux fenêtres de jugement par l'utilisation des mods DT/HT.
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+
+|  |  |
+| :-- | :-: |
+| Rupture de la rétrocompatibilité | Oui |
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:365]:
```suggestion
| Casse la rétrocompatibilité | Oui |
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+| Support du retour au mod classique | Non |
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:366]:
```suggestion
| Effet annulé par le mod Classic | Non |
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 5]
```diff
@@ -0,0 +1,445 @@
...
+| Modifié intentionnellement | Non |
+| Nécessité d'une réflexion plus approfondie | Oui |
+
+#### Le mod Flashlight n'a pas de dégradé
+
+<!-- TODO: comparison image  -->
+
+|  |  |
+| :-- | :-: |
+| Rupture de la rétrocompatibilité | Non |
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:376]:
```suggestion
| Casse la rétrocompatibilité | Non |
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 6]
```diff
@@ -0,0 +1,445 @@
...
+| Support du retour au mod classique | Non |
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:377]:
```suggestion
| Effet annulé par le mod Classic | Non |
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 7]
```diff
@@ -0,0 +1,445 @@
...
+| Modifié intentionnellement | Non |
+| Nécessité d'une réflexion plus approfondie | Oui |
+
+### Scores, points de performance et classements
+
+La notation sur lazer a été simplifiée, le principal avantage étant qu'il n'est pas nécessaire de faire un replay pour recalculer. Deux systèmes interchangeables ont été ajoutés : *standardisé* et *classique*. Le score standardisé limite le score à un maximum de 1 000 000 de points + des bonus et les multiplicateurs de score, tandis que le score classique est similaire au score standardisé, mais s'échelonne quadratiquement en fonction du nombre d'objets touchés dans une beatmap. Ces options peuvent être sélectionnées dans les paramètres, et tous les endroits où les scores sont affiché dans le jeu sont modifiés en conséquence.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:383]:
```suggestion
Le score sur lazer a été simplifié, le principal avantage étant qu'il n'est pas nécessaire de faire un replay pour le recalculer. Deux systèmes interchangeables ont été ajoutés : *standardisé* et *classique*. Le score standardisé limite le score à un maximum de 1 000 000 de points + des bonus et les multiplicateurs de score, tandis que le score classique est similaire au score standardisé, mais s'échelonne quadratiquement en fonction du nombre d'objets dans une beatmap. Ces options peuvent être sélectionnées dans les paramètres, et tous les endroits où les scores sont affichés dans le jeu sont modifiés en conséquence.
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 8]
```diff
@@ -0,0 +1,445 @@
...
+<!-- TODO: comparison image  -->
+
+|  |  |
+| :-- | :-: |
+| Rupture de la rétrocompatibilité | Non |
+| Support du retour au mod classique | Non |
+| Modifié intentionnellement | Non |
+| Nécessité d'une réflexion plus approfondie | Oui |
+
+### Scores, points de performance et classements
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:381]:
```suggestion
### Score, points de performance et classements
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 9]
```diff
@@ -0,0 +1,445 @@
...
+
+La notation sur lazer a été simplifiée, le principal avantage étant qu'il n'est pas nécessaire de faire un replay pour recalculer. Deux systèmes interchangeables ont été ajoutés : *standardisé* et *classique*. Le score standardisé limite le score à un maximum de 1 000 000 de points + des bonus et les multiplicateurs de score, tandis que le score classique est similaire au score standardisé, mais s'échelonne quadratiquement en fonction du nombre d'objets touchés dans une beatmap. Ces options peuvent être sélectionnées dans les paramètres, et tous les endroits où les scores sont affiché dans le jeu sont modifiés en conséquence.
+
+Il y a également des différences dans le score de chaque objet touché et de chaque jugement l'un par rapport à l'autre.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:385]:
```suggestion
Il y a également des différences dans le score de chaque objet et de chaque jugement l'un par rapport à l'autre.
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 10]
```diff
@@ -0,0 +1,445 @@
...
+
+Les fonctionnalités qui dépendent du score, comme le [classement par score](/wiki/Ranking#classement-par-score) et les [niveaux](/wiki/Gameplay/Score/Total_score#niveau), doivent faire l'objet de discussions plus approfondie sur la façon dont elles seront affectées.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:387]:
```suggestion
Les fonctionnalités qui dépendent du score, comme le [classement par score](/wiki/Ranking#classement-par-score) et les [niveaux](/wiki/Gameplay/Score/Total_score#niveau), doivent faire l'objet de discussions plus approfondies sur la façon dont elles seront affectées.
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 11]
```diff
@@ -0,0 +1,445 @@
...
+
+Les médailles ne sont pas attribuées. Cette fonctionnalité sera activée après la mise en œuvre des mesures anti-triche et nécessitera des réfléxions plus approfondi en ce qui concerne la modification de la difficulté de certaines médailles en raison d'autres changements dans le gameplay.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:389]:
```suggestion
Les médailles ne sont pas attribuées. Cette fonctionnalité sera activée après la mise en œuvre des mesures anti-triche et nécessitera des réflexions plus approfondies en ce qui concerne la modification de la difficulté de certaines médailles en raison d'autres changements dans le gameplay.
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 12]
```diff
@@ -0,0 +1,445 @@
...
+
+Les points de performance ne sont pas intégrés dans les classements globaux ou spécifiques aux beatmaps, mais sont plutôt intégrés dans une version séparée du site web pour l'instant : <https://lazer.ppy.sh/>.
+
+### Multijoueurs
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:393]:
```suggestion
### Multijoueur
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 13]
```diff
@@ -0,0 +1,445 @@
...
+
+#### Le mode TAG co-op n'est pas implémenté
+
+Cette implémentation est prévu, mais aucun travail n'a encore été entamé à ce sujet.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:397]:
```suggestion
Cette implémentation est prévue, mais aucun travail n'a encore été entamé à ce sujet.
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 14]
```diff
@@ -0,0 +1,445 @@
...
+
+#### Le système de trigger n'est pas implémenté
+
+![](img/sb-triggers.gif)
+
+#### Les backgrounds de beatmaps n'ont pas le bon mode de remplissage pour les storyboard 4:3
+
+![](img/sb-bg-fill-mode.jpg)
+
+Sur stable, les backgrounds de beatmap et les storyboards remplissent la hauteur de l'écran. Sur lazer, les backgrounds remplissent la largeur de l'écran, tandis que les storyboards remplissent toujours la hauteur.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:413]:
```suggestion
Sur stable, les arrière-plans et storyboards des beatmaps remplissent la hauteur de l'écran. Sur lazer, les arrière-plans remplissent la largeur de l'écran, tandis que les storyboards remplissent toujours la hauteur.
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 15]
```diff
@@ -0,0 +1,445 @@
...
+
+Cette différence entraîne des incohérences dans les modes de remplissage lorsqu'un storyboard est superposé à un background de beatmap existant.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:415]:
```suggestion
Cette différence entraîne des incohérences dans les modes de remplissage lorsqu'un storyboard est superposé à un arrière-plan de beatmap existant.
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 16]
```diff
@@ -0,0 +1,445 @@
...
+
+#### Les storyboards au format 4:3 ne sont pas recadrés au format prévu
+
+![](img/sb-crop.jpg)
+
+Certains storyboards ne sont pas conçus pour prendre en charge un format autre que 4:3, mais les storyboards sur lazer sont toujours rendus au format 16:9.
+
+## Certaines caractéristiques cosmétiques sont manquantes
+
+Il s'agit de fonctionnalités qui devraient revenir à un moment donné, mais qui ne l'ont pas fait en raison de leur faible priorité.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:425]:
```suggestion
Il s'agit de fonctionnalités qui devraient revenir à un moment donné, mais qui ne le sont pas en raison de leur faible priorité.
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 17]
```diff
@@ -0,0 +1,445 @@
...
+
+- Commentaires qui défilent sur les replays à la [Niconico](https://en.wikipedia.org/wiki/Niconico)
+- Les combo burst et les sons qui leur sont associés
+- Compte à rebours
+- Ondulations du curseur
+- Composantes non-skinnable
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:431]:
```suggestion
- Composantes non-skinnables
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 18]
```diff
@@ -0,0 +1,444 @@
...
+| Nécessité d'une réflexion plus approfondie | Oui |
+
+#### Le système de points de vie est différent
+
+Cela est principalement dû aux changements apportés au système de jugement. Par exemple, les jugements Geki et Katu n'existent pas dans lazer, ce qui a affecté les points de vie sur stable.
+
+|  |  |
+| :-- | :-: |
+| Casse la rétrocompatibilité | Oui |
+| Effet annulé par le mod Classic| Non |
```
Comment (Pisapou) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:87]:
```suggestion
| Effet annulé par le mod Classic | Non |
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 19]
```diff
@@ -0,0 +1,444 @@
...
+  - Support des versions natives de Windows 8.1+, macOS, Linux, iOS, et Android
+  - Méthode de saisie sur mobile pour tous les modes de jeu
+- Input
+  - Driver intégré pour tablette
+  - Support des contrôleurs MIDI
+  - Support des manettes de jeu
+- Interface remaniée
+  - Assistant de configuration à la première exécution
+  - Storyboards dans le menu principal
+  - Rewind F2
```
[the remark this answers is not in the document]
Comment (Pisapou) [reply wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:19]: aucun idée, et c'est pour ça que je l'ai laissé en anglais :x

[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:19]: "rewind f2" means you can press shift + f2 to select the previous random selection

File: wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md [hunk 20]
```diff
@@ -0,0 +1,444 @@
...
```
Comment (Biokax) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/fr.md:19]:
```suggestion
  - Retour en arrière pour la touche F2
```
@Pisapou je propose de le traduire de cette manière

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**